0001-typedb/03-admin/01-configuration.md

```diff
@@ -0,0 +1,248 @@
+---
+pageTitle: Configuration
+keywords: typedb, configuration, administration, config, settings
+longTailKeywords: TypeDB administration, TypeDB configuration, TypeDB settings, changing settings
+summary: TypeDB configuration guide.
+toc: false
+---
+
+# Configuration
+
+There are two ways to configure TypeDB: 
+
+* command line arguments,
+* configuration file.
+
+## Command line arguments
+
+Use command line arguments with the command `typedb server` to configure the server upon its launch. The command line 
+arguments will overwrite any relevant settings from the config file.
```

Comment:
```suggestion
arguments will override any relevant settings from the config file.
```

This is more correct, ovewrite sounds like the file is being changed, when it's not at all